packages/components/src/utils/setScrollLock.ts

```diff
@@ -10,15 +10,19 @@ export const setScrollLock = (isOpen: boolean): void => {
     body: { style, parentElement: htmlEl },
   } = document;
 
-  if (isOpen) {
-    style.top = -htmlEl.scrollTop + 'px';
-    style.overflowY = 'scroll'; // seems necessary for flyout or modal content to be scrollable
-    style.position = 'fixed';
+  if (/iPhone|iPad|iPod/i.test(navigator.userAgent)) {
+    if (isOpen) {
```

Comment:
Would be good to do deep destructuring in here since it is useless for the else case.